flink-connector-gcp-pubsub-e2e-tests/src/test/java/org/apache/flink/connector/gcp/pubsub/sink/util/PubsubHelper.java

```diff
@@ -222,4 +240,33 @@ public Publisher createPublisher(String project, String topic) throws IOExceptio
                 .setCredentialsProvider(EmulatorCredentialsProvider.create())
                 .build();
     }
+
+    public void close() {
+        if (topicClient != null) {
+            try {
+                topicClient.shutdown();
+                topicClient.awaitTermination(5, TimeUnit.SECONDS);
```

Comment:
Maybe extract this to default? If possible to Duration type

Reply:
This is only part of a test util class, we are not expecting overrides, but I would extract it since it is repeated ofc